frontend/packages/console-shared/src/components/empty-state/ConsoleEmptyState.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import {
+  EmptyStateActions,
+  EmptyStateBody,
+  EmptyStateFooter,
+  EmptyStateHeader,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  EmptyState,
+  EmptyStateProps,
+} from '@patternfly/react-core';
+import * as cx from 'classnames';
+
+export const ConsoleEmptyState: React.FC<ConsoleEmptyStateProps> = ({
+  children,
+  Icon,
+  primaryActions,
+  secondaryActions,
+  title,
+  ...props
+}) => {
+  const dataTest = props['data-test'] || 'console-empty-state';
+  const variant = props.variant || EmptyStateVariant.xs;
+  const header = (title || Icon) && (
+    <EmptyStateHeader
+      data-test={`${dataTest}-title`}
+      {...(Icon ? { icon: <EmptyStateIcon icon={Icon} /> } : {})}
+      titleText={title}
+    />
+  );
+  const bodyClassName = cx({ 'pf-v5-u-m-0': !header }); // Remove top margin if there's not a header
```

Comment:
This seems like a bug in PattenFly.

Reply:
Agreed. There is extra space above EmptyState components that do not have a header.